near-bindgen/src/environment/blockchain_interface.rs

```diff
@@ -14,6 +14,7 @@ pub trait BlockchainInterface {
     unsafe fn predecessor_account_id(&self, register_id: u64);
     unsafe fn input(&self, register_id: u64);
     unsafe fn block_index(&self) -> u64;
+    unsafe fn block_timestamp(&self) -> u64;
```

Comment:
Please expose new methods in `env.rs`. Do not wrap timestamp into `Instant` or `DateTime`, just keep it as `u64`.